joomla/joomla-cms
# [4.0] Fields redo, switcher

Pull Request for Issue https://github.com/joomla/40-backend-template/pull/395 but without transforming it to web component

### Summary of Changes
- properly initialize the element only once
- add/remove listeners correctly
- various improvements
- a11y redone

### Testing Instructions
Edit an article,
Resize the window multiple times
Make sure that there is only one switcher for the field Featured
Make sure that the data is stored correctly
Audit for a11y

### Expected result
On window resize no artefacts


### Actual result
Multiple switchers on the page


### Documentation Changes Required
